contracts/services/reverse-splitter/src/tests.rs

```diff
@@ -0,0 +1,907 @@
+use crate::msg::{
+    ActionMsgs, Config, QueryMsg, ServiceConfig, SplitAmount, SplitConfig, UncheckedSplitConfig,
+};
+use cosmwasm_std::{Addr, Decimal};
+use cw20::Cw20Coin;
+use cw_multi_test::{error::AnyResult, App, AppResponse, ContractWrapper, Executor};
+use cw_ownable::Ownership;
+use getset::{Getters, Setters};
+use valence_service_utils::{
+    denoms::{CheckedDenom, UncheckedDenom},
+    msg::ExecuteMsg,
+    msg::InstantiateMsg,
+    testing::{ServiceTestSuite, ServiceTestSuiteBase},
+};
+
+const NTRN: &str = "untrn";
+const STARS: &str = "ustars";
+const MEME: &str = "umeme";
+const CATZ: &str = "ucatz";
+const ZERO: u128 = 0u128;
+const ONE_MILLION: u128 = 1_000_000_000_000_u128;
+const TEN_MILLION: u128 = 10_000_000_000_000_u128;
+const HUNDRED_MILLION: u128 = 100_000_000_000_000_u128;
+const HALF_MILLION: u128 = ONE_MILLION / 2;
+const FIVE_MILLION: u128 = TEN_MILLION / 2;
+
+#[derive(Getters, Setters)]
+struct ReverseSplitterTestSuite {
+    #[getset(get)]
+    inner: ServiceTestSuiteBase,
+    #[getset(get)]
```

Comment:
What is the purpose of `getset(get)` again? you are building the get functions below anyways